ihatemoney/templates/forms.html

```diff
@@ -200,7 +200,11 @@
     </fieldset>
     <div class="actions">
         {{ form.submit(class="btn btn-primary") }}
-        {% if not edit %} {{ form.submit2(class="btn btn-light") }}{% endif %}
+        {% if edit %} 
+            <a href="{{ url_for(".list_bills") }}" class="btn btn-primary"> {{_("Cancel") }} </a>
```

Comment:
```suggestion
            <a href="{{ url_for(".list_bills") }}" class="btn btn-outline-secondary"> {{_("Cancel") }} </a>
```

For a better visual UX:
![image](https://user-images.githubusercontent.com/297578/166164584-030fd8c7-15ec-4733-931e-69e3e679215e.png)